downloads/index.html

```diff
@@ -1,62 +1,50 @@
+javascript: downloads.js
 ---
-
-<div id="content">
-
 <h2>Distributions</h2>
 
-<p>
-Please use the i3 packages provided by the distribution you are using. i3 is
-currently distributed in the following operating systems/distributions:
-</p>
+<p>Please use the i3 packages provided by the distribution you are using. i3 is
+currently distributed in the following operating systems/distributions:</p>
 
-<div id="distributions">
+<div class="distributions">
 {% include downloadlogo.html title="Debian GNU/Linux" link="http://packages.debian.org/sid/i3" img="/img/debian.svg" height=93 %}
 {% include downloadlogo.html title="Arch Linux" link="http://www.archlinux.org/packages/community/x86_64/i3-wm/" img="/img/archlinux.svg" height=75 %}
 {% include downloadlogo.html title="Gentoo Linux" link="http://packages.gentoo.org/package/x11-wm/i3" img="/img/gentoo-logo.svg" height=79 %}
 {% include downloadlogo.html title="Ubuntu Linux" link="http://packages.ubuntu.com/i3" img="/img/ubuntu.svg" height=75 %}
 {% include downloadlogo.html title="FreeBSD" link="http://www.freebsd.org/cgi/cvsweb.cgi/ports/x11-wm/i3/" img="/img/freebsd.png" height=59 %}
 {% include downloadlogo.html title="NetBSD" link="http://pkgsrc.se/wm/i3" img="/img/netbsd_logo.svg" height=56 %}
-<br>
 {% include downloadlogo.html title="OpenBSD" link="http://cvsweb.openbsd.org/cgi-bin/cvsweb/ports/x11/i3/" img="/img/Openbsd2.png" height=49 %}
 {% include downloadlogo.html title="openSUSE" link="http://software.opensuse.org/package/i3" img="/img/opensuse.svg" height=48 %}
 {% include downloadlogo.html title="Mageia" link="http://mageia.madb.org/package/show/application/0/name/i3" img="/img/mageia.png" height=85 %}
 {% include downloadlogo.html title="Fedora" link="https://apps.fedoraproject.org/packages/i3" img="/img/fedora.svg" height=75 %}
 {% include downloadlogo.html title="Exherbo" link="http://git.exherbo.org/summer/packages/x11-wm/i3/index.html" img="/img/exherbo.svg" height=85 %}
 {% include downloadlogo.html title="Pibang Linux" link="http://pibanglinux.com/" img="/img/pibang.png" height=61 %}
-<br>
 {% include downloadlogo.html title="Slackware" link="http://slackbuilds.org/desktop/i3/" img="/img/slackware.svg" height=75 %}
 {% include downloadlogo.html title="Qubes OS" link="https://github.com/QubesOS/qubes-desktop-linux-i3" img="/img/qubes-logo-home.svg" height=75 %}
 </div>
 
-<p>
-We also provide <a href="/docs/repositories.html">Debian and Ubuntu repositories</a> with up-to-date packages.
+<p>We also provide <a href="/docs/repositories.html">Debian and Ubuntu repositories</a> with up-to-date packages.
 </p>
 
 <h2>Downloads</h2>
 
-<p>
-  The current stable version is 4.19.
-</p>
+<p>The current stable version is 4.19.</p>
 
-<p>
-All releases are crytographically signed with the following <a href="https://gnupg.org/">GnuPG</a> key:
-<pre>
+<p>All releases are crytographically signed with the following <a href="https://gnupg.org/">GnuPG</a> key:
+
+<blockquote class="gpg-key">
```

Comment:
This blockquote shows an unnecessary horizontal scrollbar in my browser. Could you remove that please?

Reply:
11a9af9a5005fdebea71af55d4e27b000e30c905